templates/document.html

```diff
@@ -27,12 +27,11 @@
     <link rel='stylesheet' href='[%url]/stylesheets/plugin.css' />
     <link rel='stylesheet' href='[%url]/stylesheets/paginate.css' />
     <script src='[%url]/javascripts/navigation.js'></script>
-    <script src='[%url]/javascripts/plugins-search.js'></script>
     <script src='[%url]/javascripts/paginate.js'></script>
-    <script src='[%url]/javascripts/plugins.js'></script>
     [%HTML header]
     [%CSS]
     [%JAVASCRIPT]
+    <script src='[%url]/javascripts/plugins.js'></script>
```

Comment:
Can you please cross-check if this need to be `plugin-search.js`. Or maybe I am wrong that it is fine to be `plugins.js`. In any case, I would request you to please verify?

Reply:
Both need to be there -- plugins-search.js was interfering with the search function on the guides page so I added it to the metadata to be resolved in the RenderSiteTask. I moved this line because plugins-search.js needs to render first or the search on the plugins page won't work